huggingface/script/hf-ort.py

```diff
@@ -138,8 +136,8 @@
 }
 
 if not args.local_run:
-    if args.use_cu102:
-        hf_ort_env = Environment.from_dockerfile(name='hf-ort-dockerfile-10.2', dockerfile='../docker/Dockerfile-10.2')
+    if args.hf_model == 'gpt2':
+        hf_ort_env = Environment.from_dockerfile(name='hf-ort-dockerfile', dockerfile='../docker/Dockerfile_clm')
```

Comment:
why gpt2 needs a special dockerfile?

Reply:
GPT2 with fp16 requires transformers version 4.16.0 while the other models require transformers source install. The fix for that is still ongoing but should be resolved soon [Type error for GPT2 bug tracking](https://github.com/microsoft/onnxruntime/issues/11279)